components/hooks/feeds/useFeedItem.ts

```diff
@@ -1,7 +1,7 @@
 import { PostView } from "lemmy-js-client";
```

Comment:
All the changes in this file are just a result from wrapping the various functions in `useCallback`. I think an API like this might do better as a normal object which is loaded by the hook, and which uses signals or some kind of pubsub instead of the `useCallback` hell we have here. The more I work with RN over the years, the more I try to avoid hooks like this as overtime they compound hidden complexity and hidden state changes, which makes optimizations and such rather difficult.